PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -528,4 +529,6 @@ def miniAOD_customizeAllMCFastSim(process):
     process = miniAOD_customizeMETFiltersFastSim(process)
     from PhysicsTools.PatAlgos.slimming.isolatedTracks_cfi import miniAOD_customizeIsolatedTracksFastSim
     process = miniAOD_customizeIsolatedTracksFastSim(process)
+    process.patMuons.addTriggerMatching = cms.bool(False)
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.

Reply:
process.patMuons.addTriggerMatching = cms.bool(False) -> process.patMuons.addTriggerMatching = False ?

Reply:
yes